docs/release-notes/self-hosted/self-hosted-v12.0.0.md

```diff
@@ -0,0 +1,128 @@
+---
+rss_title: Codacy release notes RSS feed
+rss_href: /feed_rss_created.xml
+description: Release notes for Codacy Self-hosted v12.0.0.
+codacy_tools_version_old: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.1.6
+codacy_tools_version_new: https://github.com/codacy/codacy-tools-sh/releases/tag/sh-1.2.2
+---
+
+# Self-hosted v12.0.0
+
+These release notes are for [Codacy Self-hosted v12.0.0](https://github.com/codacy/chart/releases/tag/12.0.0){: target="_blank"}, released on June 21, 2023.<!-- TODO Update release date -->
+
+📢 [Visit the Codacy roadmap](https://roadmap.codacy.com) and <span class="skip-vale">let us know</span> your feedback on both new and planned product updates!
+
+## Upgrading Codacy Self-hosted
+
+Follow the steps below to upgrade to Codacy Self-hosted v12.0.0:
+
+1.  Check the [release notes for all Codacy Self-hosted versions](../index.md#self-hosted) **between your current version and the most recent version** for breaking changes and follow the instructions provided <span class="skip-vale">carefully</span>.
+
+1.  Follow the instructions to [upgrade your Codacy Self-hosted instance](https://docs.codacy.com/v12.0/chart/maintenance/upgrade/).
+
+1.  Update your Codacy command-line tools to the following versions:
+
+    -   [Codacy Analysis CLI 7.8.3](https://github.com/codacy/codacy-analysis-cli/releases/tag/7.8.3)
+    -   [Codacy Coverage Reporter 13.13.1](https://github.com/codacy/codacy-coverage-reporter/releases/tag/13.13.1)
+
+## Breaking changes
+
+Codacy Self-hosted v12 supports Kubernetes v1.24 and v1.25. If you're upgrading your Kubernetes installation, make sure to do the following in the recommended order to avoid service outages:
```

Comment:
Removing v1.24, as we now flag SH v11 as [already compatible with Kubernetes v1.24](https://docs.codacy.com/chart/requirements/#kubernetes-or-microk8s-cluster-setup).

```suggestion
Codacy Self-hosted v12 supports Kubernetes v1.25. If you're upgrading your Kubernetes installation, make sure to do the following in the recommended order to avoid service outages:
```